### PR TITLE
aliases can be any sequence, not only list

### PR DESCRIPTION
**Pull request by jb-leger**
When I wrote threadabilty predictor for aliases in #3086, I was considering "normal" aliases as a `list`. Today I add a aliases with a tuple, I works perfectly with xonsh, but the predictor was returning the default prediction.

This is a very minor change, I think this does not desserve a news item.

### CODE REVIEW

File: xonsh/commands_cache.py [hunk 1]
```diff
@@ -274,7 +274,7 @@ def default_predictor_alias(self, cmd0):
         alss = getattr(builtins, "aliases", dict())
         while cmd0 in alss:
             alias_name = alss[cmd0]
-            if not isinstance(alias_name, list):
+            if not isinstance(alias_name, cabc.Sequence):
```
Comment (scopatz) [inline xonsh/commands_cache.py]: should this be `if isinstance(alias_name, (str, bytes)) or not isinstance(alias_name, cabc.Sequence)`?

**Comment by scopatz** [comment]
Thanks for putting this in @jb-leger!

File: xonsh/commands_cache.py (hunk 1 above)
Comment (jb-leger) [reply xonsh/commands_cache.py]: Yes, I were right. I changed my commit.

**Comment by scopatz** [comment]
This is weird. The setup for those two tests consistently fails on windows on Python 3.7.  Maybe it is time to move test_main to a separate testing instance.

**Comment by scopatz** [comment]
Thanks @jb-leger!

**Pull request merged**